GirafRest.IntegrationTest/Tests/UserControllerTest.cs

```diff
@@ -382,14 +382,13 @@ public async void TestUserCanCheckMultipleSettings()
             Assert.Equal(60, content["data"]["timerSeconds"]);
             Assert.Equal(3, content["data"]["activitiesCount"]);
             Assert.Equal(3, content["data"]["theme"]);
-            Assert.Equal(1, content["data"]["nrOfDaysToDisplayPortrait"]);
+            Assert.Equal(2, content["data"]["nrOfDaysToDisplayPortrait"]);
```

Comment:
Hvorfor har ændret testen, så der bliver testet for to i stedet for en dag?

Reply:
I starten af sprinten stod den til 2, men efter vi lavede nogle pull requests, så var den ændret til 1, hvilket testen fejlede ved. Af den årsag ændrede vi den tilbage til 2.

Reply:
Men det ser ud til den bare skal hente settings værdierne, så den tester for det rigtige med 1. Hvis testen fejler er der fejl andetsteds

Reply:
Okay, så det skyldes at den i en test er sat til værdien 2 og nrOfDaysToDisplayLandscape er sat til værdien 5. Det er ikke deres default værdi, men i setup for den MultipleSettings test er værdierne sat til 2 og 5 - går ud fra det er meningen at testen skal teste med andre værdier end default.

Så, in conclusion, alt er fint